lib/mutant/meta/example.rb

```diff
@@ -1,7 +1,7 @@
 module Mutant
   module Meta
     class Example
-      include Adamantium, Concord::Public.new(:file, :node, :mutations)
+      include Adamantium, Anima.new(:file, :node, :node_type, :expected)
```

Comment:
> Possibly OT, but for Anima assert that all the attributes are required? I

Yes.

> I was just wondering what would happen if extra keys were provided

Exception from `Anima` namespace.

> #initialize method that uses the new hash-based args, eg:

The semantics already exist. From the call side this is indistinguishable.